plutus-core/plutus-ir/src/PlutusIR/Compiler/Names.hs

```diff
@@ -29,7 +29,10 @@ typeReplacements = replacements ++ [
     , ("(,,)", "Tuple3")
     , ("(,,,)", "Tuple4")
     , ("(,,,,)", "Tuple5")
-    ]
+    , ("(#,#)", "Tuple2")
```

Comment:
I suggest giving them a slightly different name otherwise it'll be a bit confusing (for us, mostly). Maybe `UTuple2` etc.